python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -346,6 +357,40 @@ async def configure(self, config):
                 DeprecationWarning,
             )
 
+        self.time_on_target = config.time_on_target
+        self.time_on_target = (
+            self.compute_time_on_target()
+            if config.time_on_target is None
+            else config.time_on_target
+        )
+
+    def compute_time_on_target(self):
+        """Determine the amount of time spent on target so the rotator position
+        can be optimized. Assumed slew time is 3 minutes, and that the read,
+        calculation, and offset totals 20s.
+
+        Returns
+        -------
+        self.time_on_target: float
+            Estimate of time on target in seconds.
+        """
+
+        time_on_target = 180 + self.max_acq_iter * (self.acq_exposure_time + 20)
+        if self.do_take_sequence:
+            nexp = len(self.visit_configs)
+            sci_expTimeTotal = 0.0
+            for (filt, expTime, grating) in self.visit_configs:
+                sci_expTimeTotal += expTime
+                time_on_target += sci_expTimeTotal
+
+            # assume 5s for readout and instrument reconfiguration
+            time_on_target += nexp * 5
```

Comment:
The comment says, "assume 5s for readout and instrument reconfiguration", but this is multiplying the time by 5.. do you want to add instead? 

Also, maybe make is a class attribute? `instrument_setup_overhead`? you can also use `self.latiss.read_out_time`, but since you are trying to account for more than just the readout time, maybe adding the new "overhead" attribute would be better.

Reply:
No, this is adding 5s per exposure, it's just wag but should be small in the grand scheme of things. I added brackets to make it more clear.
I don't think we're doing ourselves any favours by adding class attributes here. This is only a rough estimate.